src/Opdex.Platform.Infrastructure.Abstractions/Data/Queries/LiquidityPools/LiquidityPoolsCursor.cs

```diff
@@ -261,7 +261,7 @@ public enum LiquidityPoolOrderByType
     /// <summary>
     /// The default status of any is ignored in filter.
     /// </summary>
-    Any = 0,
+    CreatedBlock = 0,
```

Comment:
liquidity pools and transactions filter endpoints probably need a similar fix

Reply:
Believe I've covered all of the enum options which don't have a default value